index.html

```diff
@@ -0,0 +1,184 @@
+           <a href="index.html#program"> <li class="navigationItem">
+                Program
+            </li></a>
+            <a href="index.html#join"> <li class="navigationItem">
+                Join
+            </li></a>
+           <a href="about.html#sponsor"> <li class="navigationItem">
+                Sponsor
+            </li>
+        </a>
+        </a>
+           <a href="#news"> <li class="navigationItem">
+                News
+            </li>
+        </a>
+        <li class="navigationItem companyName">
+            bp Organization
+        </li>
+        </ul>
+    </div>
+<header class="header flex">
+    <h1 class="mainTitle">
+        "Mellon!"
+    </h1>
+    <h2 class="secondTitle">
+        ARE YOU READY  A JOURNEY TO  MIDDLE-EARTH?
+    </h2>
+</header>
+<main>
+    <div class="index flex">
+        <p class="description">
+            As a Tolkien reader or  a Lord of the Rings watcher, we invite you a fantastic journey in Istanbul with different people all over the world.   
+        </p>
+        <p class="date">
+            2024.03.01 (WED) - 05 (FRI)</p>
+        <p class="location">
+@Taksim Square of İstanbul in Turkey
+        </p>
+    </div>
+    <div class="program flex" id="program">
+        <h2 class="mainProgram">
+            Main Program
+        </h2>
+        <hr class="mainHr"/>
+        <ul class="schedule flex">
+            <li class="scheduleItems flex">
+                <img src="lecture-logo.png" class="scheduleItemImage">
+                <h3 class="scheduleTitle"> 
+                    Opening
+                 </h3> 
+                 <p class="scheduleIndex">
+                    As speaker grandson of J.R.R Tolkien Simon Tolkien
+                 </p>
+        </li>
+        <li class="scheduleItems flex">
+            <img src="exhibition-logo.png" class="scheduleItemImage">
+            <h3 class="scheduleTitle"> 
+                Video Shows
+            </h3>
+            <p class="scheduleIndex">
+                The highlights of series
+            </p>
+        </li>
+        <li class="scheduleItems flex">
+            <img src="workshop-logo.png" class="scheduleItemImage">
+            <h3 class="scheduleTitle">
+                
+                Workshops
+            </h2>
+            <p class="scheduleIndex">
+                Role playing games, Creating Middle-Earth musics
+            </p>
+        </li>
+        <li class="scheduleItems flex">
+            <img src="network-logo.png" class="scheduleItemImage">
+            <h3 class="scheduleTitle"> 
+                Consers
+            </h3>
+            <p class="scheduleIndex">
+                Sting,Michael Jackson-Luciano Pavarotti, Nil Karaibrahimgil, Amy Winehouse
+            </p>
+        </li>
+        </ul>
+        <button class="joinButton" id="join">
+            Get Hurry to Join Us!
+        </button>
+<a href="#program"> <p class="wholeProgram">SEE THE WHOLE PROGRAM</p></a>
+    </div>
```

Comment:
- [ ] @otmanTR great work implementing the changes requested, I noted you still haven't added font `Lato` as recommended for this design, this ensures your design text matches the Behance design also in the desktop view you added a blueline in ` See the whole Program` button which should be white,

Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/212463402-9e193a9c-fd9b-4caf-9342-011b66867ff6.png) | ![image](https://user-images.githubusercontent.com/61408860/212463417-53e5cb9d-82ff-4c9a-b024-e87015f11ca5.png)